openapi-specs/schema.json

```diff
@@ -1305,6 +1305,13 @@
       "name": "order",
       "required": false,
       "type": "string"
+    },
+    "CommonClassParameterQuery": {
+      "description": "Class parameter specifies the class from which to query objects",
+      "in": "query",
+      "name": "class",
```

Comment:
I think that we agreed on parameter's name to be `classname` not `class` am I right?

Reply:
According to the task its class not class name.
To me the classname is too verbose and doesn't add anything useful to the context.